airflow-core/src/airflow/config_templates/config.yml

```
@@ -1265,6 +1265,13 @@ traces:
       type: string
       example: ~
       default: "False"
+    otel_debug_traces_on:
+      description: |
+        If True, then traces from Airflow internal methods are exported. Defaults to False.
+      version_added: 3.0.0
```

```suggestion
      version_added: 3.1.0
```

(Or maybe 3.0.1 -- we'll need to see)

---

I set it to 3.1.0 and we can change it later if needed.